client/vscQueryParameterTools.ts

```diff
@@ -73,34 +73,15 @@ export function parseQueryForOverrides(queryText: string, language: string): Rec
  *
  * @returns a MarklogicClient based on the contents of `cfg`
  */
-export function getDbClient(queryText: string, language: string, cfg: WorkspaceConfiguration, state: Memento): ClientContext {
+export function getDbClient(queryText: string, language: string, cfg: WorkspaceConfiguration): ClientContext {
+    /** Currently not using a cache **/
```

Comment:
This comment seems unnecessary since there's no expectation that we are using a cache anywhere, right?